submissions/sofiichuk/pop-up-task/index.html

```diff
@@ -0,0 +1,52 @@
+                    <li><a href="#"><img src="../../../../popup_task/icons/mail.png" alt="mail"/><span>Mail</span></a></li>
+                    <li><a href="#"><img src="../../../../popup_task/icons/news.png" alt="news"/><span>News</span></a></li>
+                    <li><a href="#"><img src="../../../../popup_task/icons/maps.png" alt="map"/><span>Map</span></a></li>
+                    <li><a href="#"><img src="../../../../popup_task/icons/files.png" alt="files"/><span>Files</span></a></li>
+                    <li><a href="#"><img src="../../../../popup_task/icons/contacts.png" alt="contacts"/><span>Contacts</span></a></li>
+                    <li><a href="#"><img src="../../../../popup_task/icons/translation.png" alt="translate"/><span>Translate</span></a></li>
+                    <li><a href="#"><img src="../../../../popup_task/icons/videos.png" alt="videos"/><span>Videos</span></a></li>
+                    <li><a href="#"><img src="../../../../popup_task/icons/search.png" alt="search"/><span>Search</span></a></li>
+                    <li><a href="#"><img src="../../../../popup_task/icons/collection.png" alt="collection"/><span>Collection</span></a></li>
+                    <li><a href="#"><img src="../../../../popup_task/icons/apps.png" alt="apps"/><span>Apps</span></a></li>
+                    <li><a href="#"><img src="../../../../popup_task/icons/chat.png" alt="chat"/><span>Chat</span></a></li>
+                    <li>
+                        <input type="checkbox" id="input_2">
+                        <label for="input_2" class="label_2"></label>
+                        <ul class="dropdownMenu__collapsiblePart">
+                            <li><a href="#"><img src="../../../../popup_task/icons/documents.png" alt="document"/><span>Document</span></a></li>
+                            <li><a href="#"><img src="../../../../popup_task/icons/classroom.png" alt="classroom"/><span>Classroom</span></a></li>
+                            <li><a href="#"><img src="../../../../popup_task/icons/photos.png" alt="collection"/><span>Photos</span></a></li>
+                            <li><a href="#"><img src="../../../../popup_task/icons/mars.png" alt="mars"/><span>Mars</span></a></li>
+                            <li><a href="#"><img src="../../../../popup_task/icons/blogs.png" alt="blogs"/><span>Blogs</span></a></li>
+                            <li><a href="#"><img src="../../../../popup_task/icons/promo.png" alt="promos"/><span>Promos</span></a></li>
+                        </ul>
+                    </li>
+                </ul>
+            </nav>
+        </li>
+        <li class="navBar__item_img"><a href="#"><img src="../../../../popup_task/icons/bell.png" alt="notifications"/></a></li>
+        <li class="navBar__item_img"><a href="#"><img src="../../../../popup_task/icons/user.png" alt="account"/></a></li>
+    </ul>
+</nav>
+</body>
+</html>
```

Comment:
User [Prettier](https://prettier.io/) to format your code.

Reply:
Formatted.